packages/react/import-component/babel.js

```diff
@@ -41,18 +44,25 @@ module.exports = ({ types: t }) => ({
           t.assertCallExpression(argument.get('body'));
           t.assertImport(argument.get('body.callee'));
 
-          importedComponent = argument.get('body.arguments.0').node.value;
+          const { node } = argument.get('body.arguments.0');
+          importedComponent = node.value;
+          magicComments = node.leadingComments.filter(({ value }) =>
```

Comment:
Why don't you just take all comments and copy them over?

Reply:
That's a very good question! :D